Framework/PythonInterface/test/python/plugins/algorithms/MaskBTPTest.py

```diff
@@ -216,6 +216,33 @@ def test_components(self):
         masked = MaskBTP(Workspace=wksp, Components='bank3')
         self.assertEqual(10 * 90, len(masked))
 
+    def test_d33(self):
```

Comment:
Can you also check the consistency of what is expected to be masked? It seems like there is a flag on the setup of the test that switches this check off by default, but you can turn on to make sure your checks pass, then off again.